spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -3,8 +3,7 @@
 RSpec.describe Proofing::Resolution::ProgressiveProofer do
   let(:applicant_pii) { Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS }
   let(:should_proof_state_id) { true }
-  let(:ipp_enrollment_in_progress) { true }
-  let(:double_address_verification) { true }
+  let(:ipp_enrollment_in_progress) { false }
```

Comment:
Setting this to false initially will allow us to test the remote flow from the get go.